ompi/mca/io/configure.m4

```diff
@@ -14,20 +16,11 @@
 # -------------------------------------------
 AC_DEFUN([MCA_ompi_io_CONFIG],
 [
-    AC_ARG_ENABLE([mpi-io],
-        [AC_HELP_STRING([--disable-mpi-io],
-            [Disable built-in support for MPI-2 I/O, likely because
-             an externally-provided MPI I/O package will be used.
-             Default is to use the internal component system and
-             its specially modified version of ROMIO])])
-
     OMPI_MPIF_IO_CONSTANTS_INCLUDE=
     OMPI_MPIF_IO_HANDLES_INCLUDE=
     AS_IF([test "$enable_mpi_io" != "no"],
```

Comment:
on second thought, i'd rather use ```$enable_mpi_io``` everywhere, and locally define ```$define_mpi_io``` only where needed in ```ompi/mca/io/configure.m4```